irc/handlers.go

```diff
@@ -1373,6 +1373,9 @@ func kickHandler(server *Server, client *Client, msg ircmsg.Message, rb *Respons
 	if len(msg.Params) > 2 {
 		comment = msg.Params[2]
 	}
+	if comment == "" {
+		comment = client.nick
```

Comment:
`client.Nick()`

Reply:
What's the difference?

Reply:
Synchronization:

https://github.com/ergochat/ergo/blob/4785a3953a43177b22f6cc328ac870dd38877d5f/irc/getters.go#L148-L152

Technically, unsynchronized access to `(*Client).nick` is a data race, although in a context like this I think the only thing it could race against in practice is `SANICK`.